Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift

```diff
@@ -298,6 +300,28 @@ public class LiveViewCoordinator<R: RootRegistry>: ObservableObject {
         }
     }
     
+    private func getPlatformMetadata() throws -> String {
+        let encoder = JSONEncoder()
+        let metadata: [String : String] = [
+            "platform_variant": getPlatformVariant()
+        ]
+        let encodedMetadata = try encoder.encode(metadata)
+
+        return String(data: encodedMetadata, encoding: .utf8)!
+    }
+    
+    private func getPlatformVariant() -> String {
+        #if os(macOS)
+        return "macOS"
+        #elseif os(tvOS)
+        return "tvOS"
+        #elseif os(watchOS)
+        return "watchOS"
+        #else
+        return UIDevice.current.localizedModel == "iPhone" ? "iOS" : "iPadOS"
```

Comment:
The more *idiomatic* way to access this information is with `UIDevice.current.userInterfaceIdiom` 🙃 
```suggestion
        return UIDevice.current.userInterfaceIdiom == .pad ? "iPadOS" : "iOS"
```

Reply:
I should have noticed :D